sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -629,6 +630,14 @@ def __init__(self, fn):
     self._args = ()
     self._kwargs = {}
 
+  def display_data(self):
+    res = {'fn': (self.fn.__name__
+                  if hasattr(self.fn, '__name__')
+                  else self.fn.__class__),
+           'args': DisplayDataItem(str(self._args)).drop_if_default('()'),
```

Comment:
The `__call__()` logic below pulls out `label` from the first `args` index; should that be represented here?